spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'queues a job to lock the given account' do
+      other_user = FactoryGirl.create :user
+      expect(other_user).to receive(:lock_account!)
+      allow(User).to receive(:find).and_return(other_user)
+
+      post :lock_account, id: other_user.id
+    end
+  end
```

Comment:
I understood the use of the same.
I tried doing it the way it's done in "close_account" and it works fine.
https://github.com/diaspora/diaspora/pull/5643/files#diff-3212b7cc0ec103914d1d59733b84f107R38
Should I use expect method instead of this?